src/main/resources/assets/tconstruct/book/materials_and_you/sections/tools.json

```diff
@@ -4,29 +4,30 @@
     "type": "text",
     "data": "tools/repairing.json"
   },
+  
```

Comment:
I'm not sure alphabetical is the best sort order. Its better than unsorted, sure, but maybe a priority sort instead.

By priority sort, I mean an order that lists what you need the most first. So maybe pickaxe, mattock, hand axe, kama, broadsword? I'd appreciate more opinions on this one.